nautobot/extras/apps.py

```diff
@@ -44,3 +44,10 @@ def convert_field_to_list_tags(field, registry=None):
         from nautobot.extras.health_checks import DatabaseBackend
 
         plugin_dir.register(DatabaseBackend)
+
+        # Register built-in SecretsProvider classes
```

Comment:
I guess it might make the code a bit more compact, but is there any other benefit? I kind of like the imports being adjacent to their usage when we have to follow this pattern.

Reply:
Not a hill I will die on. It's just a stylistic choice really, so I am fine with whatever. Just an oddity of python really :)